### PR TITLE
login

### PR DESCRIPTION
**Pull request by Pawan918**

### CODE REVIEW

File: src/views/LoginView.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+<template>
+  <main>
+    <div class="flex justify-center items-center h-96">
+      <div class="flex justify-center items-center w-2/3 h-full  text-black">
+        <form class="flex flex-col bg-emerald-200 text-black p-20 justify-center items-center" @submit.prevent="loginHandler">
+          <div class="mb-5 flex relative">
+            <label for="username" class="w-24 block mr-5">UserName : </label>
+            <div class="relative">
+              <img src="./../assets/user-regular.svg" class="w-3 absolute  bottom-1 left-1">
```
Comment (AyushUpreti102) [inline src/views/LoginView.vue]: dont' use image here use font awesome icons components

File: src/views/LoginView.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+.form-container{
+  display: flex;
+  width: 500px;
+  justify-content: center;
+  align-items: center;
+  height: 400px;
+}
+
+</style>
```
Comment (AyushUpreti102) [inline src/views/LoginView.vue]: don't use CSS classes only tailwind classes if not using this, remove this

File: src/store/store.js [hunk 3]
```diff
@@ -11,11 +11,11 @@ const store = createStore({
...
-    setUsername (state) {
-      state.username = ''
+    setUsername (state,username) {
+      state.username = username
     },
-    setPassword (state){
-        state.password = ''
+    setPassword (state,password){
+        state.password = password
     }
```
Comment (AyushUpreti102) [inline src/store/store.js:15]: create a single mutation like eg:- setUserDetails

File: src/views/LoginView.vue [hunk 4]
```diff
@@ -38,34 +44,35 @@ export default {
     return {
       username : "",
       password : "",
+      role : "",
+      // login : true
+    }
+  },
+  computed : {
+    login(){
+      return this.$route.name === 'login' ? true : false
```
Comment (AyushUpreti102) [inline src/views/LoginView.vue]: no need to use a conditional operator, only write route === 'login' it will automatically return true or false

Comment (Pawan918) [reply src/views/LoginView.vue]: fixed it

**Pull request merged**